packages/react-native-renderer/src/ReactNativeFiberHostComponent.js

```diff
@@ -44,7 +44,9 @@ class ReactNativeFiberHostComponent {
     this._nativeTag = tag;
     this._children = [];
     this.viewConfig = viewConfig;
-    this._internalFiberInstanceHandle = internalInstanceHandle;
+    if (__DEV__) {
+      this._internalFiberInstanceHandle = internalInstanceHandle;
```

Comment:
Please include `DEV` in the name so no one is tempted to use this in prod.

Reply:
Done thanks 👍

Reply:
I still see the old name?

Reply:
I didn't save the file, pushed